synapse/pipeline.yml

```diff
@@ -323,9 +308,12 @@ steps:
   # We mostly care about testing each topology.
   # To vary the Python and Postgres versions, we use Docker images which are based
   # on an assortment of Debian releases.
```

Comment:
```suggestion
  # on an assortment of Linux distributions.
```

Reply:
Some might claim that Ubuntu ontologically fits within the umbrella of "Ubuntu releases"